docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,232 @@
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   brew install abseil cmake boost grpc protobuf xtensor pkg-config ninja buf
+   ```
+
+1. Clone the Viam C++ SDK to your computer:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   mkdir -p ~/example_workspace
+   cd ~/example_workspace
+   git clone git@github.com:viamrobotics/viam-cpp-sdk.git
+   ```
+
+1. Create a <file>build</file> directory to house the build target:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cd viam-cpp-sdk/
+   mkdir build
+   cd build
+   ```
+
+1. The Viam C++ SDK requires a specific version of `openssl` in order to build successfully.
+   Determine the version of `openssl` you are running on your macOS computer:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   brew list | grep openssl
+   ```
+
+   If the command returns `openssl@1.1`, proceed to the next step. If the command returns a different `openssl` version, install version `openssl@1.1` specifically:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   brew install openssl@1.1
+   ```
+
+1. Create an environment variable `PKG_CONFIG_PATH` which points to the version of `openssl` we will use for the Viam C++ SDK build:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   export PKG_CONFIG_PATH="/opt/homebrew/opt/openssl@1.1/lib/pkgconfig"
+   ```
+
+1. Build the C++ SDK by running the following commands:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cmake .. -DVIAMCPPSDK_USE_DYNAMIC_PROTOS=ON -DCMAKE_INSTALL_PREFIX=/usr/local -G Ninja
+   ninja all
+   sudo ninja install
+   ```
+
+   This tutorial passes two flags to the build process to help compile successfully:
+
+   - [`VIAMCPPSDK_USE_DYNAMIC_PROTOS`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md#viamcppsdk_use_dynamic_protos) to request that proto generation happen along with the build.
+   - [`CMAKE_INSTALL_PREFIX`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md#cmake_install_prefix) to install to <file>/usr/local/</file> instead of the default <file>./install</file> location.
+     To write to the restricted location <file>/usr/local/</file>, the second `ninja` invocation must also use `sudo`.
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+TODO: Linux instructions here. Currently failing at requiring tflite C headers: I have not yet been able to successfully [build the tflite headers on Linux](https://www.tensorflow.org/lite/guide/build_cmake#build_tensorflow_lite_c_library).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Download the `yamnet/classification` model file
+
+This example uses the `yamnet/classification` TensorFlow Lite model for audio classification.
+This is a pre-trained model suitable for this example, but you can also [train your own model](/manage/ml/train-model/).
```

Comment:
This is a little misleading because I don't think you actually can currently train your own model that would work with the audio classification client unless you produced something with exactly the same semantics as the yamnet/classification model. The client is pretty tied to the model in this case.

Reply:
Removed the reference to training our own here, and added to "Next steps" section instead.